sos/utilities.py

```diff
@@ -176,8 +176,8 @@ def _child_prep_fn():
             os.chdir(chdir)
 
     def _check_poller(proc):
-        if poller():
-            proc.terminate()
+        # timeout command kill child-process with SIGKILL signal
+        if poller() or proc.poll() == -9:
```

Comment:
> Last time test, I forgot to add --signal SIGKILL , so when the process is terminated , the proc.poll() equal 124.

So, the SIGKILL is unnecessary here? If that's the case, I'd prefer we keep with the default SIGTERM that timeout uses, that way the exit code handling is "normal".

Reply:
For , 'fwdownloader'  command , yes , it is unnecessary  , because 'fwdownloader' process will be killed by SIGTERM or SIGKILL, the deadlock process is its child process 'sg_map'.
I add ' --signal SIGKILL' for potential problem in the future , if a process can't be killed by SIGTERM, This change would work.

If you prefer the default SIGTERM, I will change it back in update.